spots-api/api/get-spot.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function() {
+  return function(req, res) {
+    // put json here
+    //
+    res.json({
+          "id": `${req.params.id}`,
+          "location": [xx.xx, xx.xx],
```

Comment:
this will throw an error, as xx.xx is not valid, put in numbers like 22.22, 33.33